app/domain/person/filter/attributes.rb

```diff
@@ -80,8 +80,19 @@ def attribute_condition_sql(key, value, constraint, scope)
   end
 
   def persisted_attribute_condition_sql(key, value, constraint)
+    if key == 'language' && value.present?
+      value = from_translations(value, constraint)&.keys || value
+    end
+
+    constraint = :in if value.is_a? Array
+
     sql_array = ["people.#{key} #{sql_comparator(constraint)} ?", sql_value(value, constraint)]
-    ActiveRecord::Base.sanitize_sql_array(sql_array)
+    if constraint == :in
+      query, value = sql_array
+      query.sub('?', value) # sanitize_sql_array does not work with IN queries
```

Comment:
Das öffnet sofort SQL Injection Vulnerabilities, das können wir so nicht machen. Kannst du stattdessen einen parametrisierten Query verwenden? `["people.#{key} IN (?)", value]`

Oder halt den Array Filter für den Moment wieder ausbauen?

Reply:
Ah ich verstehe jetzt warum der Array Filter. Anyways, ich habe den jetzt mal vorsorglich wieder ausgebaut, man kann jetzt halt für den Moment nur nach Sprach-Kürzeln suchen: de, fr, it, en, ...